src/boot/efi/stub.c

```diff
@@ -735,6 +884,24 @@ static EFI_STATUS run(EFI_HANDLE image) {
                            &m);
         parameters_measured = parameters_measured < 0 ? m : (parameters_measured && m);
 
+        /* Handle ucode. Always call this, even if there are no addons.
+         * This measures the .ucode section embedded in the UKI (if any). */
+        err = ucode_merge_and_measure_addons(
+                ucode_bases_addons_global,
```

Comment:
as per coding style, we indent line-broken function calls to twice the usual indentation, i.e. 16 spaces